homepage/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Noto+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Noto+Sans:700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Noto+Sans+JP:500&display=swap"
+      rel="stylesheet"
+    />
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      media="screen"
+      href="styles/main.css"
+    />
+    <title>homepage</title>
+  </head>
+  <body>
+    <nav class="navbar">
+      <a href="#" class="logo">Jenn Jiang</a>
+      <ul class="main-nav">
+        <li>
+          <a href="#" class="nav-links">Work</a>
+        </li>
+        <li>
+          <a href="#" class="nav-links">Fun</a>
+        </li>
+        <li>
+          <a href="#" class="nav-links">About</a>
+        </li>
+      </ul>
+    </nav>
+
+    <div id="main-element">
+      <h1>
+        Hello there, <br />
+        great to meet you
+      </h1>
+      <p>
+        👋🏻👋🏼👋🏽👋🏾Hi, my name is Jenn. I'm a UX Designer study at CCA.
```

Comment:
Lowercase "designer"
"studying at CCA"
also link to CCA in case someone doesn't know what it is